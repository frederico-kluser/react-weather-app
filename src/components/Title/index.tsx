import TitleContainer from './styled';

const Title = ({text}: any) => (
  <TitleContainer>{text}</TitleContainer>
);

export default Title;
