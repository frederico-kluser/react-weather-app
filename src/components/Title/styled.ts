import styled from 'styled-components';

const TitleContainer = styled.div`
  font-size: 4em;
  margin-top: 64px;
`;

export default TitleContainer;
