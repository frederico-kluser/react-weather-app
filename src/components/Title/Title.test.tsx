import renderer from 'react-test-renderer';
import Title from '.';

test('Title Renderer', () => {
  const component = renderer.create(
    <Title text="test" />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});