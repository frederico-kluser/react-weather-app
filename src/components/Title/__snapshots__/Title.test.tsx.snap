// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Renderer 1`] = `
<div
  className="sc-bdvvtL fQKQmm"
>
  test
</div>
`;
